pawno/include/modules/src/weaponshot.inc

```diff
@@ -597,6 +600,19 @@ CheckVisualDamageTextDraws()
 				Iter_SafeRemove(PlayerHasTVisualDmg, cur, i); // Safely remove him from the iterator array.
 		}
 	}
+	
+    // Update Damage Bubble
+	foreach(new i : Player)
```

Comment:
I really prefer to use y_iterate for such operations instead of a regular foreach over all players with an if statement.